hexl/ntt/ntt-default.hpp

```diff
@@ -86,8 +118,39 @@ inline void FwdButterflyRadix4(uint64_t* X0, uint64_t* X1, uint64_t* X2,
   // *X3 = ReduceMod<2>(*X3, four_times_modulus);
 }
 
-/// @brief The Harvey butterfly: assume X, Y in [0, 2q), and return X', Y' in
-/// [0, 2q) such that X' = X + Y (mod q), Y' = W(X - Y) (mod q).
+/// @brief Out-of-place Harvey butterfly: assume X_op, Y_op in [0, 2q), and 
+/// return X_r, Y_r in [0, 2q) such that X_r = X_op + Y_op (mod q), 
+/// Y_r = W(X_op - Y_op) (mod q).
+/// @param[out] X_r Butterfly data
+/// @param[out] Y_r Butterfly data
+/// @param[in] X_op Butterfly data
+/// @param[in] Y_op Butterfly data
+/// @param[in] W Root of unity
+/// @param[in] W_precon Preconditioned root of unity for 64-bit Barrett
+/// reduction
+/// @param[in] neg_modulus Negative modulus, i.e. (-q) represented as 8 64-bit
```

Comment:
Hopefully the CI build catches this, but `neg_modulus` isn't an argument here.

Reply:
Yes, the right argument should be just modulus.